### ISSUE TITLE
About training

### ISSUE THREAD

**Issue by helloful** [dseny/UHDVD#2]
Thank you very much for opening up your code. I want to ask if your model is trained on your data set and then tested on other data sets? If I want to train on the GoPro data set, what do I need to do

**Comment by helloful** [comment]
I train on the GoPro dataset and it cannot converge. The hyperparameters are all set in your code

**Comment by SamHSlva** [comment]
Have you tried the results of the pre-trained weights provided?
I am seeing a lot of horizontal stripes in outputs, both with the provided samples and my own images.

**Comment by csjunxu** [comment]
> Have you tried the results of the pre-trained weights provided? I am seeing a lot of horizontal stripes in outputs, both with the provided samples and my own images.

Is the TV loss helpful for the stripes?
